### DISCUSSION TITLE
大家對中壢的印象?

### DISCUSSION THREAD

**Discussion by v0ouo** [1101-GS4719-C/Python-Community-GS4719-C#1742] [General]
大家來到中央後，一定會對中壢這個地方感到驚訝吧!對於一個在中壢生活19年的本地人來說，已經對一些事情見怪不怪了，我自己覺得會讓外縣市奇怪的事情有幾件，首先是中壢車站前一堆外勞坐著，然後一堆人坐在前面抽菸和乞討，再來就是公車司機很多都話很多而且講話超大聲，132公車就有1個。另外就是怪人超多，像是王C銘、鄧佳華等。不知道大家對中壢的印象如何呢?
最後我要講一件事:桃園中壢一邊一國，這是不可變的事實。

**Comment by lazywulf** [comment]
結論：還是比桃園好。
中壢人簽到。

**Reply by lazywulf** [reply]
(其實是個平鎮人。

**Comment by eric202501** [comment]
作為一個曾經的武陵學生，我只能說中壢公車司機牙起來，乘客全程都在簡諧運動，還有開太認真後門忘記關的

**Reply by chia-kai-chang** [reply]
笑死~

**Reply by chia-kai-chang** [reply]
牙起來好有畫面

**Comment by alvin119** [comment]
剛來中壢三個月，第一印象真的就是外勞都充斥在火車上跟車站，每次還都跟我搶座位，聊天跟講電話也都偏大聲

**Comment by Ryanlolry** [comment]
讓人有錯覺，因為出國了，髒亂的街道，超級多的外國勞工，華而不實且超貴的異國美食，使得我對中壢的感官不是這麼好。

**Comment by THH1118** [comment]
原本以為以前的高雄火車站已經是外勞聚集的極限了，來了中壢才知道原來這個極限還有突破的空間

**Comment by WeiYen1103** [comment]
說實話，當初填大學的時候真的沒有想到中壢那麼讓我失望，首先我覺得中壢火車站超級不安全，外籍移工占滿整個街道，再者，站口外面always在施工，導致交通混亂...，也沒想到中壢真的只有sogo可以逛，沒有甚麼景點，又好潮濕，真的讓我好失望Q

**Comment by Patsy17** [comment]
交通非常混亂又沒什麼有趣的景點
重點是學校這裡好潮濕！！！

**Comment by 111426027** [comment]
沒什麼地方可以玩，而且生活機能好差。此外冬天的中央真的超冷，風大到感覺快被吹走。

**Comment by mrweixd** [comment]
中壢根本是一拳超人裡的Ｚ市，一直生怪人，都快要成立怪人協會了。

**Comment by spermwhale464** [comment]
玩的地方很少，吃也要下山，山上根本沒東西，市區高樓林立卻沒有繁華的感覺，氛圍像極了開著法拉利的乞丐QQ

**Comment by davidteng00000** [comment]
我上次星期六搭132要趕10.50的火車回家，結果公車司機在後門那一站竟然下車撿松果長達5分鐘，之後又一直跟大媽聊天開超慢，最後我11.之後才到總站，火車都錯過了，真的是超級賭爛

**Comment by 1chooo** [comment]
交通非常混亂，而且中央對外的交通都不太便利的感覺，最明顯的感覺是星期五的 GoShare 都會被大家騎走，並且連帶星期六也會沒有車可以用，要到星期天大家陸續回學校才會有車可以使用！

**Comment by Tomken123** [comment]
中壢真的很棒，要甚麼有甚麼，只是你不知道而已

**Comment by B1nn9** [comment]
如果喜歡大自然的話，推薦可以多去青埔晃晃，到處都是公園綠地，而且人也相對市區少很多，很常能在公園碰見在地居民遛可愛的狗勾，他們有些甚至會很熱情邀請你來摸狗勾，有緣的話還可以因為這樣認識😆

**Comment by chuanmin0317** [comment]
做為一個南部人，我覺得中央的真的蠻冷的，而且天氣變化也有點大，中壢的話，我覺得真的沒有什麼地方好逛的，不過去台北很快，如果想出去玩的話蠻方便的，這點就還不錯

**Comment by sorn32** [comment]
我真的沒想過都要畢業了，火車站前面還是一直在施工，到底是怎麼做到搞這麼久的(^_^)
然後冬天宵夜街口風大到有夠無言，真的超級討厭

**Comment by chiuwangwang** [comment]
我覺得桃園中壢一邊一國，這是確實是不可辯的事實，身為住在其他都市的人，很難覺得從主要市區到另一個市區風景很少有市區鄉間市區的繁華差異，通常都是市區鬧區市區，我每次中壢桃園跑，都讓我有很大的差異感，時常覺得是不是走錯路。

**Comment by YANRONG0123** [comment]
對於一個離島人來說，中壢讓我感覺到車非常多，無論是機車或是汽車都相當明顯，但是感覺交通設計方面並沒有與數量成正比，連食物方面也蠻慘的。

**Comment by woohsin** [comment]
來之前一直聽說中壢跟桃園不一樣，來了之後我是沒有感覺到不一樣啦哈哈。
但是前幾天發現一件有趣的事情，用google map的話，火車站附近超多店的分類是寫色情按摩店，密度很高，但是這真的不會被查嗎？

**Comment by wanderer657** [comment]
身為一個在中壢生活了19年的本地人，我以為自己已經對這個地方的種種司空見慣。但有一天，一場超現實的奇遇改變了我的看法。

那是一個普通的午後，我獨自走在市中心的街道上，突然間，一陣強烈的震動使整個地面都在搖晃。我猛然抬頭，發現一個巨大的漩渦從天而降，它在城市上空盤旋，帶來一股強烈的能量。

在漩渦的中心，出現了一個光芒閃爍的門戶，我不禁被它的神秘吸引。毫不猶豫地走向門戶，我感到一股奇異的力量將我吸入其中。

當我重新打開眼睛時，我發現自己置身於一個完全陌生的世界。這裡的建築物充滿著未來主義的風格，人們穿著奇特的服裝，行走在流光溢彩的街道上。我意識到，我進入了一個與現實完全不同的超現實空間。

在這個世界裡，我遇到了各種各樣的人物和生物。有著超凡能力的英雄，有著邪惡陰謀的反派，還有與我一同探險的夥伴。我們一起挑戰著未知的考驗，探索著這個神秘世界的奧秘。

我們穿越了荒蕪的沙漠，遠涉危險的森林，甚至冒險探索了一個被遺忘已久的古老城市。在這些冒險中，我們遭遇了各種挑戰和危險，但我們始終保持著勇氣和堅持，一路向前。

最終，我們來到了這個世界的核心，一個充滿力量和能量的神秘之地。在那裡，我們面對著一場生死關頭的決戰，必須用盡一切力量來對抗黑暗勢力的威脅。

雖然這場冒險充滿了困難和挑戰，但我們最終戰勝了所有的障礙，帶著滿懷的回憶和豐富的經歷返回了現實世界。即使這段冒險只是一場夢，但它對我來說卻是一段難以忘懷的奇遇，讓我更加珍惜生活中的每一刻。

**Comment by thomasxshen** [comment]
我只能說對於一個背井離鄉來到中壢的台北人來說，中壢真的是很爛，交通設計一塌糊塗，車站附近髒亂不堪，還有一堆怪人會來搞你心態，且警察的執法態度及效率也令人不敢恭維，一個地下停車場屬於那個派出所的轄區都搞不明白，叫個警察還要等個兩三小時，感覺中壢警察大概只會拿警棍打人吧！附近菜市場攤販直接擺到路地方政府也是擺爛不想管，每天就是看到車跟人緊緊貼著不出車禍才怪，中壢市中心即使高樓林立，也無法讓人感受到這座城市的繁華，對比於正在高速開發的桃園市，無不令人感到悲哀啊！

**Comment by hbh10601** [comment]
曾經的武陵學生路過，我住桃園區，但先不戰桃園中壢，我只想問一個問題，
那就是"到底為什麼不往後走"?!!!
每次放學要上車都擠不上去，可是都看到後面一堆空間沒站人不知道在幹嘛，超氣......。

**Comment by willychiuchiu** [comment]
有一說一，其實中壢生活便利性算是非常高的，基本上要什麼有什麼，還有公車、火車、高鐵甚至捷運，對住在大台北地區的我並不會有太大的不適應，只是這地區的人種真的太複雜，導致交通、治安等等都比較差，容易讓人有很像是個尚未開發地帶的形象。

**Comment by Mitochondrionjessica** [comment]
中壢火車站周圍是體驗東南亞異國風情的好地方，走在街上是撲鼻而來的香氣，逛街買東西幾乎是聽不懂的語言。除此之外還有最具有特色的交通，亂成一團是基本盤，偶爾還能體驗招手公車司機不想讓你上車不停的失落，或是公車開到一半突然熄火的小驚喜(有一次從中壢公車站坐回中央，路上熄火四次，第一次全部旅客倒抽一口氣，到了第四次大家都習慣了^^)。覺得生活太平淡無趣的朋友，請推薦他們來中壢玩一趟，肯定有所收穫。

**Comment by 20050201Shawn** [comment]
上學期開學對中壢的印象就是交通亂成一團，生活機能沒到多好，髒髒黑黑汙染嚴重破破爛爛。但經歷這幾天地震覺得，其實中壢天災沒那麼嚴重，可能都點在人禍上面了哈哈。

**Comment by LinChloe** [comment]
第一次從北部來看到中壢火車站前的場景，真的嚇傻。
外籍勞工的香水味，躺在座椅的流浪漢，永遠被圍著的大馬路。
這是火車大站嗎，我心裡覺得到台灣的貧民窟了。

**Comment by happy246810** [comment]
對於中壢火車站附近有很多外籍勞工，我覺得沒有很意外，因為在台北車站跟板橋車站也有這個情況，只是因為板車跟北車空間比較大，所以都會移到內部聚集。還有請加開132跟133，不然每次搭車人潮都像是在逃難。

**Comment by EPHMR0524** [comment]
這裡根本是台灣佛州，很多奇怪的人，但最讓我覺得不習慣的果然還是交通。常常有人車騎到一半就停下來東張西望，害我差點撞上去，最誇張 的是有一次我晚上騎在中正路，有人給我倒著開。我還以ˋ為是我騎太快，真的是點點點

**Comment by yyujimmy** [comment]
對於中壢這個地方的印象確實因人而異，對於一些生活在當地的人來說，可能已經習以為常，而對於外地來的人可能會感到新奇或驚訝。中壢作為一個交通樞紐，確實吸引了不少外勞和過路的人，這也使得中壢的街頭文化更加多樣化。

至於公車司機的風格和行為，每個地方的公車司機可能都有各自的特色，有些司機可能講話大聲，但也有些司機可能比較溫和。而關於怪人的存在，每個城市都會有一些獨特的人物，他們或許是某個社區的傳奇，也可能是城市的一部分文化。

不過，中壢作為桃園的一部分，確實有其獨特的地位和風貌，無論是車站前的景象還是街頭文化，都是中壢這個城市的一部分。每個地方都有自己的特色和魅力，這也是城市多樣性的魅力所在。

**Comment by KH1126** [comment]
中壢的交通非常混亂，而且中央對外的大眾交通班次也太少。
但我對中壢印象最深刻的地方是外籍勞工是真的多。

**Comment by jim1743** [comment]
中壢對我得印象就是車很多，每次搭公車，在下班的尖峰時刻，到達火車站的時間都很久，另外火車站前的外籍勞工真的很多，還以為來到國外，另外感覺中壢的房子特別老舊，相比其他都市，中壢對我的觀感來說沒有很好。

**Comment by cassy0201** [comment]
第一次出中壢火車站的時候真的有嚇到，之前就聽別人說出站像出國，身歷其境才發現有多誇張，雖然尊重吸菸的權力，但每次出站聞到濃厚的煙味真的很不舒服，印象不是太好

**Comment by tshsu04224** [comment]
我對中壢的印象自從看完富都青年之後，就一直覺得如果要拍台灣版富都青年，中壢車站絕對是首選拍攝取景地，那個場景真的超級像！另外說到公車，有一次搭132下山吃飯，按了下車鈴結果司機在講電話沒聽到，直接往前多開兩站，我大喊之後才停下來，甚至是急煞直接把我隨便丟在路邊根本不是站牌的那種，真的嚇個半死，只能說中壢公車真的每次搭都要先做好心理準備。

**Comment by 155Julia** [comment]
作為一個中部人，我覺得中央大學的氣候真的蠻冷的，尤其是天氣變化很大，時常讓人措手不及。至於中壢這個地方，我覺得實在沒什麼好逛的地方，餐飲價格也偏高。不過，中壢離台北非常近，這是一大優勢。台北常常有各種活動，只需要短時間的車程就能到達，非常方便。對於喜歡外出玩樂或參與活動的人來說，這確實是中壢的一大亮點。儘管中壢本身可能缺乏娛樂設施，但方便的交通使得它成為一個不錯的居住選擇。

**Comment by yubo0oo** [comment]
中壢的交通相當糟糕
路線雜亂無章
用路人完全不手法
堪稱行人地獄
